module/x/gravity/types/msgs.go

```diff
@@ -289,40 +307,44 @@ func (msg MsgCancelSendToEthereum) GetSigners() []sdk.AccAddress {
 	return []sdk.AccAddress{acc}
 }
 
-// NewMsgEthereumHeightVote returns a new MsgEthereumHeightVote
-func NewMsgEthereumHeightVote(ethereumHeight uint64, signer sdk.AccAddress) *MsgEthereumHeightVote {
-	return &MsgEthereumHeightVote{
-		EthereumHeight: ethereumHeight,
-		Signer:         signer.String(),
+// NewMsgEVMHeightVote returns a new MsgEVMHeightVote
+func NewMsgEVMHeightVote(chainID uint32, ethereumHeight uint64, signer sdk.AccAddress) *MsgEVMHeightVote {
+	return &MsgEVMHeightVote{
+		EvmHeight: ethereumHeight,
+		Signer:    signer.String(),
+		ChainId:   chainID,
 	}
 }
 
 // Route should return the name of the module
-func (msg MsgEthereumHeightVote) Route() string { return RouterKey }
+func (msg MsgEVMHeightVote) Route() string { return RouterKey }
 
 // Type should return the action
-func (msg MsgEthereumHeightVote) Type() string { return "ethereum_height_vote" }
+func (msg MsgEVMHeightVote) Type() string { return "ethereum_height_vote" }
```

Comment:
"ethereum_height_vote" -> "EVM_height_vote"

Reply:
corrected in a few places in this file